crc_errors/CRC_Count_check.py

```diff
@@ -28,50 +25,43 @@
 class common_setup(aetest.CommonSetup):
     """ Common Setup section """
 
-    # CommonSetup have subsection.
-    # You can have 1 to as many subsection as wanted
-
     # Connect to each device in the testbed
     @aetest.subsection
     def connect(self, testbed):
-        genie_testbed = Genie.init(testbed)
+        genie_testbed = load(testbed)
         self.parent.parameters['testbed'] = genie_testbed
         device_list = []
         for device in genie_testbed.devices.values():
             log.info(banner(
-                "Connect to device '{d}'".format(d=device.name)))
+                f"Connect to device '{device.name}'"))
             try:
                 device.connect()
             except Exception as e:
-                self.failed("Failed to establish connection to '{}'".format(
-                    device.name))
+                self.failed(f"Failed to establish connection to '{device.name}'")
 
             device_list.append(device)
 
         # Pass list of devices the to testcases
         self.parent.parameters.update(dev=device_list)
+        print(self.parent.parameters)
```

Comment:
hmm i would either remove this, or put it in `logger.debug()`

Reply:
Oops I left this in. It was only me debugging